packages/ndla-ui/src/User/UserInfo.tsx

```diff
@@ -75,6 +85,18 @@ export const UserInfo = ({ user }: Props) => {
           </li>
         ))}
       </InfoList>
+      {parsedUser.grepCodes.length > 0 && (
+        <InfoList data-hj-suppress>
+          <li key="grepCodes">
+            {t('user.groupTypes.grepCode')}
+            <InfoList>
+              {parsedUser.grepCodes.map((code) => (
+                <li key={code.id}>{code.displayName}</li>
```

Comment:
Kunne kanskje hatt med code.code i parantes bak displayname. Veit i alle fall at Guttorm gjerne vil vise fram grep for å bygge bevissthet rundt disse.